Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.Core/Common/BussinessRules/BusinessRuleValidator.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace EvolutionaryArchitecture.Fitnet.Contracts.Core.Common.BussinessRules;
```

Comment:
Why not to moveit to a building blocks package (common) which we have in chapter 4? It can be just method with another signature, next to today's validate

Reply:
It easier for me in next PR extract code to commons, Merge it then remove code from contracts project and update refereneces pointing to common